hangar-api/pom.xml

```diff
@@ -49,6 +49,12 @@
 			<version>19.0</version>
 		</dependency>
 
+        <dependency>
```

Comment:
Ah, looks better now, may want to view the PR using `?w=1` to see the actual changes though. Code quality check is failing due to `configure()` being empty inside `HangarInjector` - got any candidates for DI? I could do the one I spotted, it might make the PR fairly large though.